websites/A/Averge/metadata.json

```
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Averge.EU",
```

The service field should match the folder the activity is in
```suggestion
    "service": "Averge",
```